### ISSUE TITLE
Packed executable crashes with 3.92 but works with 3.91

### ISSUE THREAD

**Issue by dacto** [upx/upx#39]
#### Updates:
* Also broken for the i386 variant of the source executable.
* Attached i386 files.
## What's the problem (or question)?

Packed  executable crashes

## What should have happened?

Packed executable should print out
```
Hello World

```

## Do you have an idea for a solution?

- Source executable packed with UPX 3.91 works properly.
- Source executable packed with UPX 3.92 crashes.

## How can we reproduce the issue?

Attached [upx3.92_app_crash.zip](https://github.com/upx/upx/files/655188/upx3.92_app_crash.zip) contains:
```
1. unpacked source executable                        -> upxtest_original.exe
2. source executable packed with UPX 3.91 (no flags) -> upxtest_upx3.91.exe
3. source executable packed with UPX 3.92 (no flags) -> upxtest_upx3.92.exe
```
1. pack `upxtest_original.exe` from [upx3.92_app_crash.zip](https://github.com/upx/upx/files/655188/upx3.92_app_crash.zip)  (no flags)
2. run packed executable and witness crash

## Please tell us details about your environment.

* UPX version used (`upx --version`): upx 3.92
* Operating System and version: Windows 7 Pro
* CPU architecture: amd64
* Attachments:
  * x86_64: [upx3.92_app_crash.zip](https://github.com/upx/upx/files/655188/upx3.92_app_crash.zip)
  * i386: [upx3.92_app_crash_i386.zip](https://github.com/upx/upx/files/655362/upx3.92_app_crash_i386.zip)

**Comment by ml1050** [comment]
Fixed in c51864ca39a1fbb9d417227bf8dc7ecafa1b94e5. Thanks for you report. Can we use your example program in the UPX test suite?

**Comment by dacto** [comment]
I appreciate you addressing the issue.
Yes, you may use my example program in the UPX test suite.
